tachyon/zk/air/plonky3/base/lagrange_selectors.h

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2022 The Plonky3 Authors
+// Use of this source code is governed by a MIT/Apache-2.0 style license that
+// can be found in the LICENSE-MIT.plonky3 and the LICENCE-APACHE.plonky3
+// file.
+
+#ifndef TACHYON_ZK_AIR_PLONKY3_BASE_LAGRANGE_SELECTORS_H_
+#define TACHYON_ZK_AIR_PLONKY3_BASE_LAGRANGE_SELECTORS_H_
+
+namespace tachyon::zk::air::plonky3 {
+
+template <typename T>
```

Comment:
```suggestion
template <typename F>
```
How about F instead of T since it's zk stuff?

Reply:
I think it would be better as the generic `T`, since Lagrange selectors are used in a multitude of instances. More specifically, so far,`T` is used as type `F`, `ExtField`, or `std::vector<F>`.